dev/test/specs/starter-test.js

```diff
@@ -380,4 +382,40 @@ describe('test Arcs demo flows', function() {
 
     browser.close();
   });
+
+  it('can use an arc with the default global manifests', function() {
+    initTestWithNewArc();
+
+    // remove solo from our URL to use the default
+    const url = new URL(browser.getUrl());
+    url.searchParams.delete('solo');
+    browser.url(url.href);
+
+    // load our utils in the new page
+    loadSeleniumUtils();
+
+    waitForStillness();
+    browser.waitUntil(
+      () => {
+        const allSuggestions = pierceShadows([
+          'div[slotid="suggestions"]',
+          'suggestion-element'
+        ]);
+        if (!allSuggestions.value || 0 == allSuggestions.value) {
+          return false;
+        }
+
+        // we hit at least a single suggestion, good enough!
+        return true;
+      },
+      5000,
+      `couldn't find any suggestions; this might indicate that a global manifest failed to load`
+    );
+
+    // treat the fact that we found any suggestions as a good enough
+    // indication that there aren't any major issues with globally available
+    // manifests.
+
+    browser.close();
```

Comment:
i don't think this needs to call browser.close() because that was for closing the new arc tab in the other test cases.

Reply:
It was opening a tab the same as the others.. but, having explicit closes was lame. I've added code to close at the start.